src/Compiler/Checking/CheckDeclarations.fs

```diff
@@ -957,6 +957,32 @@ module MutRecBindingChecking =
                         | definition -> 
                             error(InternalError(sprintf "Unexpected definition %A" definition, m)))
 
+                // Report any desugared properties
+                if defnAs.Length > 1 then
```

Comment:
Do I get it right that this refers to "get" and "set" here? If so, can we actually have `= 2` here instead of `> 1`?